compute_rouge_scores.py

```diff
@@ -0,0 +1,59 @@
+from src.data_utils import get_gold_xsum_data, load_summaries_from_logs, load_xsum_dict
+from sumtool.storage import get_summaries
+from src.metrics import rouge
+from tqdm import tqdm
+import numpy as np
+
+
+if __name__ == "__main__":
+    gold_sums, gold_metadata = get_gold_xsum_data()
+
+    MODEL_RESULTS = {
+        "BartGEF": load_summaries_from_logs(
+            "results/fbs-logs/bart-full-classifier-knnv1.json", max_iterations=5
+        ),
+        # "pegasus_gef": load_summaries_from_logs(
+        #     "results/fbs-logs/bart-full-classifier-knnv1.json", max_iterations=5
+        # ),
```

Comment:
to be commented in once we have a pull FBS gen for Pegasus?

Reply:
Yes